tools/buildsys/src/builder.rs

```diff
@@ -590,14 +607,6 @@ where
         .filter(|e| e.is_file() || e.is_symlink())
 }
 
-/// Retrieve a BUILDSYS_* variable that we expect to be set in the environment,
-/// and ensure that we track it for changes, since it will directly affect the
-/// output.
-fn getenv(var: &str) -> Result<String> {
-    println!("cargo:rerun-if-env-changed={}", var);
```

Comment:
It's really just this one `getenv` function that emits `rerun-if-changed`, which is why you picked up some new variables for tracking.